components/AppFooter.vue

```diff
@@ -115,6 +133,11 @@ export default {
       ARCANA_DISCORD_URL,
     }
   },
+  methods: {
+    handleBookDemo() {
+      window.open(process.env.ARCANA_BOOK_DEMO_URL, '_blank')
```

Comment:
We should start preferring `a` elements over calling `window.open`. Possible to use this markup instead?

```
<a :href="..." target="_blank">
  <v-button ... />
</a>
```